CHANGELOG.md

```diff
@@ -0,0 +1,139 @@
+-   Venia components are now portable and can be used in any React application.
+-   New centralized driver and adapter concepts for all PWA-Studio component input/output.
+-   Configurable low level UI elements, such as lists and forms, available for developers without the need for drivers or adapters.
+-   Improved error handling to help detection and recoverability.
+
+#### Related documentation updates
+
+-   [Modular Venia components][] ([#796][])
+-   [Venia directory structure][]([#487][] [#752][])
+
+### App-like experience
+
+-   Pagination feature added to Venia reference storefront for app-like navigation.
+-   Inline checkout implemented in the Venia reference storefront.
+-   Basic account creation and management implemented in Venia.
+-   Loading state implemented to improved the app-like experience.
+
+#### Related documentation updates
+
+-   [Venia Checkout implementation][] ([#673][])
+
+### Improved shopper experience
+
+-   Braintree checkout integration in the Venia reference storefront.
+-   Search with intelligent previews implemented in the Venia reference storefront.
+
+#### Related documentation updates
+
+-   [Braintree integration][]([#793][])
+
+### PWA compliance
+
+-   Web App Manifest file created for the Venia reference storefront.
+    This file provides the metadata for adding the Venia storefront to a mobile home screen.
+-   Service worker optimizations
+
+### Performance enhancements
+
+-   Smart caching implemented to improve service worker catalog caching
+-   Bandwidth usage optimizations achieved using:
+    -   Dynamic image compression/resize
+    -   Advanced minification
+    -   Bundle splitting
+-   Improved app performance on devices, including smoother renders and faster [Time To First Meaningful Paint][].
+
+### GraphQL usage
+
+-   GraphQL usage increased to match coverage updates in Magento 2.3.0.
+
+### Magento Cloud compatibility
+
+-   A new Magento module (`Magento_UpwardConnector`) is available to route frontend requests through an UPWARD-PHP server.
+-   Solution Implementers(SI) can replace existing Magento cloud storefronts with a new PWA storefront using the UPWARD connector module.
+-   Documentation created for deploying storefront to Magento Cloud
+-   Assistance offered by Magento Enterprise Cloud deployment team
+
+#### Related documentation updates
+
+-   [README][] for the `Magento_UpwardConnector` module, which contains Cloud deployment instructions.
+
+### Other documentation updates
+
+#### New topics
+
+-   [Theme vs Storefront][] ([#509][])
+-   [Install Venia sample data][] ([#474][])
+-   [Peregrine Routing][] ([#453][])
+
+#### Updated topics
+
+-   [Certificate and Valet+ troubleshooting][] ([#637][])
+-   [Tools and Libraries][] ([#621][])
+
+## Known issues
+
+-   After submitting a successful order, users are unable to add items to the cart. ([#916][])
```

Comment:
This is not exactly true. You can add items to the cart, it just takes a second. The real problem is that the app throws up an error notification. The user has to actively click out of the error notification. The error is a false positive.

Reply:
Description updated